### PR TITLE
Remove loop needed to assure distinct random accounts in bank demo

### PR DESCRIPTION
**Pull request by nvb**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/3330)

### CODE REVIEW

File: sql/bank_test.go [hunk 1]
```diff
@@ -83,12 +83,9 @@ CREATE TABLE IF NOT EXISTS bank.accounts (
...
 			from := rand.Intn(*numAccounts)
-			var to int
-			for {
-				to = rand.Intn(*numAccounts)
-				if from != to {
-					break
-				}
+			to := rand.Intn(*numAccounts - 1)
+			if from == to {
+				to = *numAccounts - 1
```
Comment (tamird) [inline sql/bank_test.go]: pedantic: this makes `*numAccount - 1` twice as likely as other choices

Comment (nvanbenschoten) [inline sql/bank_test.go]: Are you sure? We decreased the domain of the `rand.Intn` to [0, *numAccounts - 1), so *numAccounts - 1 will only be used when `from == to`.

Comment (tamird) [inline sql/bank_test.go]: Ah, I see. LGTM

Comment (tamird) [inline sql/bank_test.go]: Actually, what if `from` == `*numAccounts - 1`?

Comment (nvanbenschoten) [inline sql/bank_test.go]: Then there's no chance `from == to`, so the `to := rand.Intn(*numAccounts - 1)` will give exactly what we want. It's kind of the best possibility.

**Pull request merged**